conf/cvoc/cvoc-conf.json

```diff
@@ -0,0 +1,131 @@
+[{
```

Comment:
When converting docs to .rst we should make both JSON examples downloadable from the docs. We can also inline them if that's easier to avoid duplicating the same JSON in the docs themselves.